Regression_CF.R

```diff
@@ -0,0 +1,69 @@
+#setwd("C:/Users/frase/Documents/Rclass/Regression-repo")
+
+read.csv("iris.csv")
```

Comment:
YOu need to save the data object! 

iris <- read.csv("iris.csv")

It works here because itʻs a built in datafile, but if it werenʻt it would not work